Add example for Series.str.slice()

avelineg opened the pull request:
Add example to show that negative values for start are allowed for Series.str.slice().

- [ X] closes #27802
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Commit:
Add example for Series.str.slice()

Add example to show that negative values for start are allowed for Series.str.slice().

pep8speaks commented:
Hello @avelineg! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`pandas/core/strings.py`](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py):

> [Line 1445:1](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py#L1445): [E101](https://duckduckgo.com/?q=pep8%20E101) indentation contains mixed spaces and tabs
> [Line 1445:1](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py#L1445): [W191](https://duckduckgo.com/?q=pep8%20W191) indentation contains tabs
> [Line 1446:1](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py#L1446): [W191](https://duckduckgo.com/?q=pep8%20W191) indentation contains tabs
> [Line 1447:1](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py#L1447): [W191](https://duckduckgo.com/?q=pep8%20W191) indentation contains tabs
> [Line 1448:1](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py#L1448): [W191](https://duckduckgo.com/?q=pep8%20W191) indentation contains tabs
> [Line 1449:1](https://github.com/pandas-dev/pandas/blob/c6f651b17169679be8de675ff099ab8b29183d99/pandas/core/strings.py#L1449): [W191](https://duckduckgo.com/?q=pep8%20W191) indentation contains tabs

Commit:
Add example for Series.str.slice()

Add example to show that negative values for start are allowed for Series.str.slice().

topper-123 commented:
Thanks, @avelineg
